src/main/resources/META-INF/mods.toml

```diff
@@ -14,7 +14,7 @@ displayURL="https://age-series.org/"
 # A file name (in the root of the mod JAR) containing a logo for display
 logoFile=""
 credits="The Age Series Team"
-authors="jrddunbr, Caeleron, Grissess, Baughn, empireu, DarthSidiousPalpatine, AlanFletes, bloxgate"
+authors="jrddunbr, Caeleron, Grissess, Baughn, DarthSidiousPalpatine, AlanFletes, bloxgate"
```

Comment:
_writes code change. also removes self from authors list_

Seriously, dude. You wrote more of the Eln2 mod than any of the people in this list except maybe Grissess.